### ISSUE TITLE
Prusa Link doesn't preview QOI images in the web interface.

### ISSUE THREAD

**Issue by Claghorn** [prusa3d/Prusa-Link#899]
I changed the slicer setting from PNG to QOI in order to get thumbnail previews on my MK4 LCD screen (which did work), but now if I connect to PrusaLink with google chrome the web interface doesn't display the thumbnail.

**Comment by wesseli01** [comment]
I got the same Problem.

**Comment by Claghorn** [comment]
I took a look at the latest configuration for Prusa Slicer and I see the "official" G-code thumbnails setting is
`16x16/QOI, 313x173/QOI, 440x240/QOI, 480x240/QOI, 640x480/PNG` which seems to give me thumbnails everywhere again.

**Comment by TojikCZ** [comment]
That is correct. PrusaLink will not do QOI. Just leave a PNG in the slicer. QOI is a RAM saving format for the 32 bit anyways, in MK3 PrusaLink it has no advantages that I am aware of. If you are talking about the MK4/XL PrusaLink, then that belongs to the prusa3d/Prusa-Firmware-Buddy repo

**Comment by JamesRHarris** [comment]
This appears to be a dupe of https://github.com/prusa3d/Prusa-Link-Web/issues/475

**Comment by TojikCZ** [comment]
Well prusa3d/PrusaLink does not support the newer buddy based printers at all.

So yea, it's a duplicatein a wrong repository. Thanks closing
